apps/site/src/views/Resources/components/ApiGroup/ApiGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from "./ApiGroup.module.scss";
+import { API_Group_Props } from "../../interfaces/interfaces";
+export default function API_group({
+	title,
+	description,
+	stickerSrc,
+	tagSrc,
+	tagLink,
+	postItSrc,
```

Comment:
Nitpick/Chore: Post-it is a company trademark. "sticky note" might be a more appropriate term.